src/cocoa/toga_cocoa/app.py

```diff
@@ -33,9 +33,9 @@ def applicationOpenUntitledFile_(self, sender) -> bool:
         for filetype in self.interface.document_types:
             fileTypes.addObject(filetype)
 
-        NSDocumentController.sharedDocumentController().runModalOpenPanel(panel, forTypes=fileTypes)
+        NSDocumentController.sharedDocumentController.runModalOpenPanel(panel, forTypes=fileTypes)
```

Comment:
I'm guessing this is needed for High Sierra compatibility; in which case, there will also be a change needed in libs to mark this particular API as a known class property. See the handling of `NSApplication.sharedApplication` for the analogous handling.

Reply:
Great catch, thanks, I have now added the class property.